pages/agent/v3/securing.md

```diff
@@ -41,6 +41,11 @@ Command line evaluation can be disabled by setting [`no-command-eval`](/docs/age
 >🚧 Custom hooks
 > If you have a custom <code>command</code> hook, using <code>no-command-eval</code> will have no effect on your command execution. See <a href="#allowing-a-list-of-plugins">Allowing a list of plugins</a> and <a href="#customizing-the-bootstrap">Custom bootstrap scripts</a> for examples of how to completely lock down your agent from arbitrary code execution.
```

Comment:
Switch to using Markdown syntax, which [can be used in callouts](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts) as doing so makes the source content more readable.

```suggestion
> 🚧 Custom hooks
> If you have a custom `command` hook, using `no-command-eval` will have no effect on your command execution. See [Allowing a list of plugins](#allowing-a-list-of-plugins) and [Custom bootstrap scripts](#customizing-the-bootstrap) for examples of how to completely lock down your agent from arbitrary code execution.
```